src/celeritas/ext/VecgeomParams.hh

```diff
@@ -38,6 +41,11 @@ class VecgeomParams
     // Construct from a GDML filename
     explicit VecgeomParams(const std::string& gdml_filename);
 
+#ifdef CELERITAS_USE_GEANT4
+    // Create a VecGeom model from a pre-existing Geant4 geometry
+    VecgeomParams(const G4VPhysicalVolume* p_G4world);
+#endif
```

Comment:
```suggestion
    // Create a VecGeom model from a pre-existing Geant4 geometry
    VecgeomParams(const G4VPhysicalVolume* p_G4world);
```

Reply:
We don't want the class definition to change based on compile options. This is one of the things that has been a thorn in VecGeom's side.